sklearn/datasets/openml.py

```diff
@@ -395,6 +395,10 @@ def fetch_openml(name=None, version='active', data_id=None, data_home=None,
     cache : boolean, default=True
         Whether to cache downloaded datasets using joblib.
 
+    return_X_y : boolean, default=False.
+        If True, returns ``(data, target)`` instead of a Bunch object.
+            See below for more information about the `data` and `target` object
```

Comment:
No extra indentation is needed here (the "See .." should align with "If True ...")

object -> objects